### DISCUSSION TITLE
Install mold on ubuntu 20.04

### DISCUSSION THREAD

**Discussion by divn123** [rui314/mold#1360] [General]
i tried to install via apt
```
$ sudo apt install mold
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package mold
```
but seems like mold is not included as package for ubuntu 20.04. 
how should install mold for ubuntu 20.04 . is it possible to install it using ppa?

**Comment by rui314** [comment]
I don't know how you can install mold to a specific Linux distribution. You may want to ask in the ubuntu forum rather than here.

In any case, you can always download a binary from https://github.com/rui314/mold/releases if you don't want to build it yourself.

**Reply by divn123** [reply]
thank you for the reply. i will ask in ubuntu forum
